infrastructure/main.tf

```diff
@@ -53,12 +53,14 @@ locals {
 }
 
 module "db" {
-  source              = "git@github.com:hmcts/moj-module-postgres.git?ref=master"
+  source              = "git@github.com:hmcts/moj-module-postgres?ref=cnp-449-tactical"
   product             = "${var.product}-db"
   location            = "${var.location_db}"
   env                 = "${var.env}"
-  postgresql_database = "postgres"
-  postgresql_user     = "letter_tracking"
+  database_name       = "send_letter"
+  postgresql_user     = "send_letter"
+  sku_name            = "GP_Gen5_2"
```

Comment:
Does the sku name needs to be this ? or can we have a proper name?

Reply:
I think this is the actual name for "General purpose" sku